spec/libraries/aws_engine_spec.rb

```diff
@@ -75,6 +75,25 @@ def initialize
       end
     end
   end
+
+  module EC2
```

Comment:
__[Reek]__

[UncommunicativeModuleName] Aws::EC2 has the name 'EC2'


<!-- Comment by [SideCI](https://sideci.com) -->